pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -420,6 +402,10 @@ def on_validation_end(self, trainer, pl_module):
             if self.last_model_path and self.last_model_path != filepath:
                 self._del_model(self.last_model_path)
 
+    def _is_valid_monitor_key(self, metrics):
+        is_valid = self.monitor in metrics or len(metrics) == 0
+        return is_valid
```

Comment:
can be inlined. what happens if `metrics` is empty? it seems like this is fine if save_top_k <= 0? 
```suggestion
        return self.monitor in metrics or len(metrics) == 0
```